src/MenuGraphr.ts

```diff
@@ -598,12 +604,22 @@ export class MenuGraphr implements IMenuGraphr {
             }
         }
 
-        menu.selectedIndex = selectedIndex;
```

Comment:
There's no longer any real need to keep `selectedIndex` as a variable up above, since you don't always use it here.